packages/grid/x-grid-modules/src/models/gridOptions.tsx

```diff
@@ -276,18 +276,14 @@ export interface GridOptions {
  * The default [[GridOptions]] object that will be used to merge with the 'options' passed in the react component prop.
  */
 export const DEFAULT_GRID_OPTIONS: GridOptions = {
-  autoHeight: false,
   rowHeight: 52,
   headerHeight: 56,
   scrollbarSize: 15,
   columnBuffer: 2,
-  enableMultipleSelection: true,
-  enableMultipleColumnsSorting: true,
```

Comment:
I wonder if we will need to include the default value back for the automatic generation of the documentation, we will see.

Reply:
Probably not 🤔